app/screens/navigation.ts

```diff
@@ -499,6 +499,10 @@ export function goToScreen(name: AvailableScreens, title: string, passProps = {}
 
     DeviceEventEmitter.emit(Events.TAB_BAR_VISIBLE, false);
 
+    if (NavigationStore.getScreensInStack().includes(name)) {
+        Navigation.pop(name);
+    }
```

Comment:
To confirm, I guess this means we can't stack screens more than once?

Reply:
You are correct.